### ISSUE TITLE
Unable to alter blog views

### ISSUE THREAD

**Issue by illegalnumbers** [jipiboily/monologue#297]
It seems that the blog views can't be altered when you copy over the views from the engine unless you modify the engine directly. For example, I wanted to add the 'btn' class on to the link with class of 'button' to get my styling. Since your layout yields to Rails I'm not sure I can get these to change and reflect appropriately since the mounted engines views are being taken instead of mine.

Or it might be midnight and I'm making a mistake somewhere.

**Comment by illegalnumbers** [comment]
An easy fix is to just move the monologue views to the correct rails engine layout that would match the right path Rails needs to resolve when you generate views. ie. instead of generating under the blog/ directory, just move them to views/monologue. Then you won't have to have that layout check either in the config initializer and the views will work out of the box.

**Comment by jipiboily** [comment]
Yup, you're right, that's where they go. The generator needs to be fixed I guess.
